source/whats-new.txt

```diff
@@ -28,8 +28,31 @@ Learn what's new in:
 What's New in 4.6
 -----------------
 
-See `v4.6.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v4.6.0>`__
-on GitHub.
+New features of the 4.6 Node.js driver release include:
+
+TypeScript: ``ChangeStreamDocument``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Ability to define your own custom type for the top level document
+  returned in a ``change`` event.
+- Improvements on the type information of the default
+  ``ChangeStreamDocument`` to default to union of the possible events.
+- Added the ``to``, ``lsid``, ``txnNumber`` and ``clusterTime`` properties.
+
+.. note:: 
+
+   If you want to defer the type corrections, use the watch API as
+   follows:
+
+   .. code-block:: typescript
+
+      // 'X' controls the change document type
+      .watch<any, X>()
```

Comment:
Issue: I'm not able to figure out what this note applies to.

Reply:
So the drivers RN mention this as something users need to do if they want to upgrade to this driver version but defer type corrections. It looked/sounded important so I brought it over, but am open to removing it if its not important.